### PR TITLE
Combine everything together for future pulls

### PR DESCRIPTION
**Pull request by weikiat97**

### CODE REVIEW

**Comment by sheryltay28** [comment]
Minor bugs but looks good!!

File: src/main/java/seedu/address/logic/commands/AddClassroomCommand.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.classroom.Classroom;
+
+
+/**
+ * Adds a classroom to the notebook.
+ */
+public class AddClassroomCommand extends Command {
```
Comment (SebastianLie) [inline src/main/java/seedu/address/logic/commands/AddClassroomCommand.java:14]: Really well written class!

File: src/main/java/seedu/address/logic/parser/AddClassroomCommandParser.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+import seedu.address.model.classroom.Classroom;
+
+
+/**
+ *
+ * Parses input arguments and creates a new AddClassroomCommand object
+ */
+public class AddClassroomCommandParser implements Parser<AddClassroomCommand> {
+
+    /**
```
Comment (SebastianLie) [inline src/main/java/seedu/address/logic/parser/AddClassroomCommandParser.java:19]: Perhaps can consider making the javadocs more succinct!

File: src/main/java/seedu/address/logic/parser/SetClassroomCommandParser.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+
+
+/**
+ * Parses input arguments and creates a new SetClassroomCommand object
+ */
+public class SetClassroomCommandParser implements Parser<SetClassroomCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the SetClassroomCommand
```
Comment (SebastianLie) [inline src/main/java/seedu/address/logic/parser/SetClassroomCommandParser.java:21]: Can consider making this comment more succinct too!

File: src/main/java/seedu/address/logic/commands/AddClassroomCommand.java (hunk 1 above)
Comment (weikiat97) [reply src/main/java/seedu/address/logic/commands/AddClassroomCommand.java:14]: Thanks!

**Comment by SebastianLie** [comment]
Great work friends! Really well written code! :)

File: src/main/java/seedu/address/logic/commands/AddClassroomCommand.java (hunk 1 above)
Comment (Xelyion) [reply src/main/java/seedu/address/logic/commands/AddClassroomCommand.java:14]: Looks good

**Pull request merged**